src/api/examples.md

```diff
@@ -58,7 +58,7 @@ There are five types of contexts, **Client Context**, **Non-Networked**, **Stati
 - Scripts can spawn objects inside a static context.
 - Scripts run on both the server and the client.
 - Useful for things reproduced easily on the client and server with minimal data (procedurally generated maps).
-    - Send a single networked value to synchronize the server and clientsâ€™ random number generators.
+    - Send a single networked value to synchronize the server and clients random number generators.
```

Comment:
This still needs to be `client's` - the `s is necessary because we're talking about random number generators that belong to the client.